build.project.xml

```diff
@@ -250,14 +244,15 @@
     <!-- Compile SASS to CSS. -->
     <target name="compile-sass"
             description="Compile the SASS to CSS.">
-        <!-- Make sure the CSS directory exists. -->
+                <!-- Make sure the CSS directory exists. -->
         <mkdir dir="${sass.output.dir}" />
         <sass
-            executable="${sass.bin}"
-            flags="--sourcemap=${sass.sourcemap}"
-            style="${sass.style}"
-            input="${sass.input}"
-            output="${sass.output.dir}/${sass.output.file}" />
+                executable="${sass.bin}"
+                flags="--sourcemap=${sass.sourcemap}"
+                style="${sass.style}"
+                input="${sass.input}"
+                output="${sass.output.dir}/${sass.output.file}"
+                failonerror="false" />
```

Comment:
I would not allow this to fail, since it will cause the website to be unstyled.

Reply:
The reason that we are allowing this to fail is that in the docker-compose version, sass is being compiled by a different container. This will fail in the web container because sass executable will not be found.

Reply:
The command is still needed for local development. If we ditch local development support entirely, we can remove that one too.

Reply:
Dropping support for local development is not on the table. This change breaks this target, so let's remove this change and simply not execute this target.